### PR TITLE
Feature/structure update

### PR DESCRIPTION
**Pull request by protsack-stephan**
Updated tests structure for readability.

### CODE REVIEW

File: page_html_test.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	srv := httptest.NewServer(createHTMLServer())
+	defer srv.Close()
+
+	client := NewClient(srv.URL)
+	client.options.PageHTMLURL = htmlTestURL
+
+	html, status, err := client.PageHTML(context.Background(), htmlTestTitle)
+
+	if status != http.StatusOK {
+		t.Fatal("html response error")
```
Comment (sashah2) [inline page_html_test.go:42]: Let's show a difference:

E.g.: `html response error: expected 200; got 404`

File: page_html_test.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	}
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if string(html) != htmlTestBody {
+		t.Fatal("html responses don't match")
```
Comment (sashah2) [inline page_html_test.go:50]: The same here ^. Maybe we can create a helper like `compare(expected, got)`

Comment (sashah2) [reply page_html_test.go:50]: Please update it for every tests

**Review by sashah2 — COMMENTED** [review]
Good work!

I have left a couple of comments about the tests clean. 👍

File: page_html_test.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	router := http.NewServeMux()
+
+	router.HandleFunc(htmlTestURL+htmlTestTitle, func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(htmlTestBody))
+	})
+
+	router.HandleFunc(htmlTestURL+htmlTestTitle+"/"+strconv.Itoa(htmlTestRevision), func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(htmlTestBody))
```
[the remark this answers is not in the document]
Comment (protsack-stephan) [reply page_html_test.go:28]: I'll do a separate PR for this, need to thing about it more.

**Pull request merged**